asf_heat_pump_suitability/utils/geo_utils.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import geopandas as gpd
+import shapely
+
+
+def transform_gdf_drop_duplicates(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
```

Comment:
The reason we don't just drop_duplicates here and drop duplicate representative points instead is because there were some geometries in the files which were nearly identical but not exactly the same. We want to drop identical and almost identical overlapping land parcels/ building footprints.

See comment below - this assumes identical polygons will always generate the same representative points. Is this correct?

Reply:
Yes, I traced where representative point comes from (geopandas representative Point -> shapely point on surface -> geos interior point area) and found a description of the algorithm: https://libgeos.org/doxygen/classgeos_1_1algorithm_1_1InteriorPointArea.html
My reading of this is that it is deterministic and you should get the same result every time for the same input polygon.